src/components/ChannelPreviewMessenger.js

```diff
@@ -81,15 +81,36 @@ export const ChannelPreviewMessenger = themed(
       this.props.setActiveChannel(this.props.channel);
     };
 
+    renderAvatar = (otherMembers) => {
+      const { channel } = this.props;
+      if (channel.data.image)
+        return <Avatar image={channel.data.image} size={40} />;
+
+      if (otherMembers.length === 1)
+        return <Avatar image={otherMembers[0].user.image} size={40} />;
```

Comment:
maybe show something if there's more than 2 people in the channel as well.

Reply:
Yeah I was thinking, if there are moer than 2 people in conversation ... then user should just set the channel image. Because its not really a one on one conversation anymore